### ISSUE TITLE
[DataCap Application] <NCBI> - <Sequence Read Archive>

### ISSUE THREAD

**Issue by Philoooo9** [Destore2023/MetaPathways-Bookkeeping#24]
### Version

1

### DataCap Applicant

Philoooo9

### Project ID

x

### Data Owner Name

National Library of Medicine

### Data Owner Country/Region

United States

### Data Owner Industry

Life Science / Healthcare

### Website

https://www.nlm.nih.gov/

### Social Media Handle

@NLM_NIH

### Social Media Type

Twitter

### What is your role related to the dataset

Data Preparer

### Total amount of DataCap being requested

12PiB

### Expected size of single dataset (one copy)

1592.4TiB

### Number of replicas to store

8

### Weekly allocation of DataCap requested

512TiB

### On-chain address for first allocation

f1mp4xe27l46wl5vq4qdmedj5k5qm7twsewbgkgvy

### Data Type of Application

Public, Open Dataset (Research/Non-Profit)

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
The Sequence Read Archive (SRA), produced by the National Center for Biotechnology Information (NCBI) at the National Library of Medicine (NLM) at the National Institutes of Health (NIH), stores raw DNA sequencing data and alignment information from high-throughput sequencing platforms. The SRA provides open access to these biological sequence data to support the research community's efforts to enhance reproducibility and make new discoveries by comparing data sets.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
Public SRA data in the original (user submitted) format from select high value and newly-released studies as well as all public-access SRA formatted ETL+BQS data. Also included is all SRA metadata that can be leveraged for attribute-based data discovery.
```

### Where was the data currently stored in this dataset sourced from

AWS Cloud

### If you are a data preparer. What is your location (Country/Region)

Malaysia

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
lotus is our mainly way to prepare data.
```

### Please share a sample of the data

```text
aws s3 ls --no-sign-request s3://sra-pub-src-1/
aws s3 ls --no-sign-request s3://sra-pub-src-2/
aws s3 ls --no-sign-request s3://sra-pub-run-odp/
aws s3 ls --no-sign-request s3://sra-pub-metadata-us-east-1/sra/metadata/
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Daily

### For how long do you plan to keep this dataset stored on Filecoin

More than 3 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America, Europe

### How will you be distributing your data to storage providers

HTTP or FTP server, Shipping hard drives, Lotus built-in data transfer

### How did you find your storage providers

Slack, Partners

### Please list the provider IDs and location of the storage providers you will be working with.

```text
f03216485/Sichuan
f03214920/HongKong
f03228358/Zhejiang
f01975299/Guangdong
f01084941/Hong Kong
```

### How do you plan to make deals to your storage providers

Lotus client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by Destore2023** [comment]
Please fill in this form to complete your information. https://www.wenjuan.com/s/qAVFfuN/

**Comment by Philoooo9** [comment]
![image](https://github.com/user-attachments/assets/8f440fd8-5dca-453b-a6c3-ade4548fbe1e)
Can you see my form?

**Comment by Destore2023** [comment]
@Philoooo9 Good. Got it.
![image](https://github.com/user-attachments/assets/318b6733-bffe-416c-809d-aa9c4b528de2)

**Comment by Destore2023** [comment]
There used to have someone who tried to store this dataset. Why do you want to store the dataset?

When you want to use datacap to store data, you'd better to select sps in different regions. Also, please make your sps support retrieval! We will check it from time to time.

**Comment by Philoooo9** [comment]
@Destore2023 We have stored this dataset for 2 years. Due to the large size of this dataset, our local storage space is becoming insufficient to keep this dataset all the time. We want to store this dataset into filecoin network. We just want to keep the part we have downloaded. It is from NLM and useful to research. Could we store this data here?

**Comment by Destore2023** [comment]
@Philoooo9 Filecoin welcomes all kinds of valuable data being stored. But please take the storage process seriously and use datacap in a careful way. We will check cid report from time to time.

**Comment by Philoooo9** [comment]
Thanks allocator. We will actively communicate with sps and you! @Destore2023 Please let us try our best.

**Comment by Philoooo9** [comment]
Dear allocator @Destore2023, we'd add three sps into our plan. 
f01084413 Hunan
f03100009 Guangdong
f03233966 Zhejiang

**Comment by Destore2023** [comment]
checker:manualTrigger

**Comment by Destore2023** [comment]
@Philoooo9 Seems there's one sp do not support retrieval yet. Please remind this sp or change sp who is supporting retrieval. Also, please add sps to store more replicas.

**Comment by Philoooo9** [comment]
@Destore2023 Thank you allocator. Thanks for your tips.
We contacted with that sp. They found out it was a network problem and are fixing it.

**Comment by Destore2023** [comment]
checker:manualTrigger

**Comment by Philoooo9** [comment]
![image](https://github.com/user-attachments/assets/bb53a7ed-01e4-4a50-aaf2-515bac95e7e7)
@Destore2023 allocator, this sp is repairing the node and it supports retrieval in spark again. Please approve us next round.

**Comment by Destore2023** [comment]
@Philoooo9 Happy to see the improvement.

**Comment by Philoooo9** [comment]
@Destore2023 New sps in our list.
f03100004 Guangdong
f03231666 Sichuan
f02816097 HongKong

**Comment by Philoooo9** [comment]
@Destore2023 New sps in our list.
f03099981 HongKong
f02851143 Luzhou

**Comment by Destore2023** [comment]
@Philoooo9 Hello, can you specified more precisely what data you plans to store?

**Comment by Philoooo9** [comment]
@Destore2023 
Hi allocator, the data we stored is from aws s3 ls --no-sign-request s3://sra-pub-src-1/.
We have downloaded 1592.4TiB data in total. The dataset is including data as below.
SRZ190764/mESCs_Mettl14_WT_fast5.tar.gz.1
SRZ190765/mESCs_Mettl3_KO-ref_fast5.tar.gz.1
SRZ190766/mESCs_Mettl3_WT_fast5.tar.gz.1
SRZ190767/SC_rRNA_dcDNA_fast5.tar.gz.1
SRZ190768/SC_rRNA_dRNA_fast5.tar.gz.1
SRZ190769/SC_YPD_heatshock_fast5.tar.gz.1
SRZ190801/SC_YPD_heatshock_ctrl_fast5.tar.gz.1
...

Now our plan is running, please support us. Thank you.

**Comment by Destore2023** [comment]
@Philoooo9 Got it. Please ensure retrieval rate and update sps in time.

**Comment by Destore2023** [comment]
checker:manualTrigger

**Comment by Destore2023** [comment]
@Philoooo9 There're 3 sps showed low retrieval results.
f03100009
f03228358
f03233966
Please check it.

**Comment by Philoooo9** [comment]
@Destore2023 Thank you for your comments. We stopped to send data to these sps because they stopped joining our plan. Can we invite new sps to continue our plan?

**Comment by Destore2023** [comment]
@Philoooo9 Considering the situation, I think you should bring in new sps to work with.

**Comment by Philoooo9** [comment]
@Destore2023 Thank you! We will find more sps to finish the storage. For now, we start with other sps that we were already working with previously, and when we find new sps, we'll send data to them.
Please allocate us another round.

**Comment by Destore2023** [comment]
Based on your situation, I'm going to sign 1.5PiB this round.

**Comment by Philoooo9** [comment]
@Destore2023 Hi allocator, we have found new sps and here's all sps we are working together.
f03099987 Bangkok
f03066836 Changsha
f03251993 Ningbo
f01660795 Guangdong
f01084413 Singapore
f03100004 Guangdong
f03231666 Sichuan
f02816097 Hong Kong
f03099981 Hong Kong
f02851143 Sichuan

> @Destore2023 Thank you for your comments. We stopped to send data to these sps because they stopped joining our plan. Can we invite new sps to continue our plan?

**Comment by Destore2023** [comment]
checker:manualTrigger

**Comment by Destore2023** [comment]
@Philoooo9 As per our previous discussion, excluding the three sps that are no longer cooperating, all the other sps are supporting retrieval. But it seems that the retrieval rate has decreased a bit. Can you explain about it?
>![image](https://github.com/user-attachments/assets/1badbaee-ab5b-486b-bb02-2f2e3c126191)

**Comment by Philoooo9** [comment]
@Destore2023 
Hi allocator, we are keeping an eye on this. After communicating with sps, we confirm that the decrease is temporary. We are contacting them to make improvements. It will increase in 2-3 days.

**Comment by Philoooo9** [comment]
@Destore2023 Since we are currently working with 10 sps, can we change our application to store 10 copies?

**Comment by Destore2023** [comment]
@Philoooo9 Yes, you should follow the changes in retrieval rates and keep them at high levels. 
ok. If you want to add replicas, you also need to change the request amount of datacap.

**Comment by Destore2023** [comment]
@Philoooo9 Since your sps need improvement, this round I'll allocate you half of the amount of our allocation schedule - 1.875P. If your retrieval rate improves, I'll return to the proper allocation percentage based on your progress.

**Comment by Philoooo9** [comment]
@Destore2023 Thank you allocator, we have a very good experience from your advice.

**Comment by Philoooo9** [comment]
@Destore2023 Hello allocator, since there was a problem with the retrieval of one sp, we added a new sp to help us store. f01431043 Hongkong

**Comment by Destore2023** [comment]
checker:manualTrigger

**Comment by Destore2023** [comment]
@Philoooo9 You should check this report and make changes to your distribution plan and sp selection as soon. Retrieval has improved a bit and you should look for more improvement.

**Comment by Philoooo9** [comment]
@Destore2023 Thank you allocator, I'll make adjustments as soon as I can.

**Comment by Destore2023** [comment]
checker:manualTrigger

**Comment by Destore2023** [comment]
@Philoooo9 
Last time
![image](https://github.com/user-attachments/assets/39487ea8-38ad-46b7-870b-e4aa7086317f)
This time
![image](https://github.com/user-attachments/assets/ac272b8a-9398-4c97-b420-e31abb7d64d9)

Retrieval has improved, but you should focus your distribution.

**Comment by Philoooo9** [comment]
@Destore2023 Thank you allocator, we are trying to make better distribution. It need some time to wait for the distribution percentage changing.

**Comment by Philoooo9** [comment]
@Destore2023 Hello allocator, we added two sps into our plan. 
f03325538 Hong kong f01313 Hong kong
Please check our report and allocate more datacap to us.

**Comment by Destore2023** [comment]
checker:manualTrigger

**Comment by Destore2023** [comment]
@Philoooo9 Bad report. 
Retrieval has improved a little. But your distribution still has problem.
Also your replicas are more than your original plan, it seems that you didn't have good way to use datacap to store data.
We will not allocate more datacap to you unless you can solve the problem and improve your technology.

**Comment by Destore2023** [comment]
@Philoooo9 Please provide explanation about it.
>Most of the data is stored in a single copy, even though 10 copies were declared. In some cases, data is replicated up to 15 times.

**Comment by Philoooo9** [comment]
@Destore2023 Hello allocator. When we work with sps, we would send the first part of our data to sps and see how they store and make data retrievable. We stopped sending more data when some of the sps had problems retrieving or storing it. That's why it happened.

> [@Philoooo9](https://github.com/Philoooo9) Please update your sp list in your application directly (e.g., via an edit) to reflect any changes. Deactivated SPs should be clearly marked (for example, using a strikethrough)

We will update it soon!



> [@Philoooo9](https://github.com/Philoooo9) Bad report. Retrieval has improved a little. But your distribution still has problem. Also your replicas are more than your original plan, it seems that you didn't have good way to use datacap to store data. We will not allocate more datacap to you unless you can solve the problem and improve your technology.

This is also caused by our cooperation with sps. The reason is the same as the answer of first problem.

Please be assured that we will continue to improve and enhance our storage plan!

**Comment by Destore2023** [comment]
@Philoooo9 Also, can you provide information to connect stored files with their original sources?

**Comment by Philoooo9** [comment]
@Destore2023 Hello allocator, this is the proof which can connect our storage with our original sources.

<img width="955" alt="Image" src="https://github.com/user-attachments/assets/8cf98c68-8027-4853-b3b7-c4ceebd1b74e" />

<img width="1036" alt="Image" src="https://github.com/user-attachments/assets/eb00b253-fdd2-4ac2-b982-21d0a226d50f" />

![Image](https://github.com/user-attachments/assets/2275f7bc-01b7-4fbe-9b93-2c60cf5b58a8)

**Comment by Destore2023** [comment]
@Philoooo9 I noticed you have used up datacap, will you continue storing data?

**Comment by Philoooo9** [comment]
@Destore2023 Yes allocator, we are waiting for it.

**Comment by Destore2023** [comment]
@Philoooo9 You have to make sure you don't add more replicas next. And retrieval is still important for you to improve. Do you agree to make upgrade?

> [@Destore2023](https://github.com/Destore2023) Hello allocator. When we work with sps, we would send the first part of our data to sps and see how they store and make data retrievable. We stopped sending more data when some of the sps had problems retrieving or storing it. That's why it happened.
> 
> > [@Philoooo9](https://github.com/Philoooo9) Please update your sp list in your application directly (e.g., via an edit) to reflect any changes. Deactivated SPs should be clearly marked (for example, using a strikethrough)
> 
> We will update it soon!
> 
> > [@Philoooo9](https://github.com/Philoooo9) Bad report. Retrieval has improved a little. But your distribution still has problem. Also your replicas are more than your original plan, it seems that you didn't have good way to use datacap to store data. We will not allocate more datacap to you unless you can solve the problem and improve your technology.
> 
> This is also caused by our cooperation with sps. The reason is the same as the answer of first problem.
> 
> Please be assured that we will continue to improve and enhance our storage plan!

**Comment by Philoooo9** [comment]
@Destore2023 Yes, we are willing to follow the rules and try the best. Thank you for your trust.

**Comment by Destore2023** [comment]
@Philoooo9 I'll allocate you half of the amount of our allocation schedule - 1.875P. The next step is up to you.

**Comment by Philoooo9** [comment]
@Destore2023 Thank you allocator!

**Comment by Destore2023** [comment]
We noticed that gov team mentioned in slack that there was a problem with ipni, which would affect the retrieval results shown in the report. We will keep one eye on their progress. Please keep your retrieval available.

**Comment by Philoooo9** [comment]
@Destore2023 Please check and give us next round of datacap.

**Comment by Destore2023** [comment]
checker:manualTrigger

**Comment by Destore2023** [comment]
checker:manualTrigger

**Comment by Destore2023** [comment]
@Philoooo9 I think you should double-check your distribution program. Your replica count has grown again.
